File: functions/clean.py
import re
with open("index.js", "r", encoding="utf-8", errors="replace") as f:
    content = f.read()
lines = content.split("\n")
clean = []
for line in lines:
    if len(line) > 300:
        line = re.sub(r'"[^"]{80,}"', '"[msg]"', line)
        line = re.sub(r"`[^`]{80,}`", "`[msg]`", line)
    clean.append(line)
result = "\n".join(clean)
with open("index_clean.js", "w", encoding="utf-8") as f:
    f.write(result)
print("Done. Size:", len(result)//1024, "KB")

File: functions/clean.js
const fs = require("fs");
const content = fs.readFileSync("index.js", "utf8");
const lines = content.split("\n");
const clean = lines.map(line => {
  if (line.length > 300) {
    return line.replace(/"[^"]{80,}"/g, '"[msg]"').replace(/`[^`]{80,}`/g, "`[msg]`");
  }
  return line;
});
const result = clean.join("\n");
fs.writeFileSync("index_clean.js", result, "utf8");
require("fs").appendFileSync("clean_done.txt", "Size: " + result.length + "\n");
